app/src/main/java/com/breadwallet/BreadApp.java

```diff
@@ -12,29 +14,16 @@
 import com.breadwallet.tools.util.LocaleHelper;
 import com.breadwallet.tools.util.Utils;
 import com.google.firebase.crashlytics.FirebaseCrashlytics;
+import com.loafwallet.util.cryptography.KeyStoreKeyGenerator;
+import com.loafwallet.util.cryptography.KeyStoreManager;
 
-import com.google.android.gms.ads.identifier.AdvertisingIdClient;
-import com.google.android.gms.common.GooglePlayServicesNotAvailableException;
-import com.google.android.gms.common.GooglePlayServicesRepairableException;
```

Comment:
I am using optimize import on Android Studio, that's why this is removed

Reply:
To be clear @andhikayuana , importing in the IDE reduced compile time and reduce binary size?  Do we need to set our AS Env to match yours?

Reply:
> importing in the IDE reduced compile time and reduce binary size? 

actually no, it's only for readability. 
I do manually before commit using theses shortcut/menu

![Screenshot 2024-12-13 at 17 05 35](https://github.com/user-attachments/assets/ed3cf6ce-90b8-4eeb-be81-5b7526c46ef4)
![Screenshot 2024-12-13 at 17 05 18](https://github.com/user-attachments/assets/a95d56f5-3bf8-4998-bc23-4cd9a772943d)